path_login.go

```diff
@@ -95,11 +95,26 @@ func (b *kubeAuthBackend) pathLogin(ctx context.Context, req *logical.Request, d
 		return nil, errors.New("could not load backend configuration")
 	}
 
+	if err := validateAliasNameSource(role.AliasNameSource); err != nil {
```

Comment:
Even though I see the good intent of having validation on `role.AliasNameSource` in here, this can actually cause errors on older entries that existed before this change since that value is not present for those roles.

Reply:
I believe that the following update to `role()` mitigates this potential issue:

https://github.com/hashicorp/vault-plugin-auth-kubernetes/blob/23ce29fe0a96069475e173c17039b39ab8b7e18c/backend.go#L145-L147

Reply:
addressed in https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/110/commits/40ecafed3570dcfd269833c7834264757cbed356